Corona-Warn-App/src/main/res/layout/home_statistics_cards_sevendayrvalue_layout.xml

```diff
@@ -4,11 +4,12 @@
     xmlns:tools="http://schemas.android.com/tools">
 
     <androidx.constraintlayout.widget.ConstraintLayout
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
+        android:layout_width="300dp"
```

Comment:
Why the fixed values?

Reply:
They are fixed, so all cards have the same size. I put them here so the preview is correct. Should move them to dimens though.